src/modules/app/containers/Switcher.js

```diff
@@ -136,52 +122,44 @@ export class Switcher extends React.Component<PropsType> {
     throw new Error(`For the route ${currentRoute} was no content selected in the Switcher which should be displayed.`)
```

Comment:
Just a grammar thing: I think it should be "No content was selected for route ${currentRoute} to be displayed" or something similar :)